dockerfiles/cpubuilder/build_manylinux_x86_64.Dockerfile

```diff
@@ -57,6 +57,7 @@ RUN yum install -y epel-release && \
     yum install -y gcc-toolset-12-libatomic-devel && \
     yum install -y patchelf && \
     yum install -y vim-common git-lfs && \
+    yum install -y libgfortran5 && \
```

Comment:
`libgfortran5` is the runtime library which _should_ be installed already (?). At least the image has the compiler package `gfortran` (logs says `/opt/rh/gcc-toolset-12/root/usr/bin/gfortran` is used for compilation) and thus the runtime library should be there. Thus you might face a different issue.